packages/origin-ui-core/src/utils/exchange/types.ts

```diff
@@ -13,6 +13,7 @@ export interface IProductDTO {
     generationFrom?: string;
     generationTo?: string;
     gridOperator?: string[];
+    externalDeviceId?: { id: string; type: string };
```

Comment:
Replace with the proper type `IExternalDeviceId`.

https://github.com/energywebfoundation/origin/blob/52e5fccdeb303133b3db9413fc317e4a1c60ebca/packages/origin-backend-core/src/Device.ts#L10